common/src/main/res/values/strings.xml

```diff
@@ -742,8 +742,12 @@
     <string name="sensor_name_last_app">Last Used App</string>
     <string name="websocket_setting_name">Websocket Settings</string>
     <string name="websocket_setting_summary">Manage when we are in direct communication with your server via websockets.</string>
-    <string name="websocket_setting_description">Please select when you would like the application to attempt to directly communicate with your Home Assistant instance for all communication.  This will include push notifications, if you are on a minimal build this needs to be always to consistently get push notifications.</string>
+    <string name="websocket_setting_description">Please select when you would like the application to attempt to directly communicate with your Home Assistant instance for all communication. This will include push notifications, if you are on a minimal build this needs to be always to consistently get push notifications.</string>
     <string name="websocket_setting_never">Never</string>
     <string name="websocket_setting_while_screen_on">While Screen On</string>
     <string name="websocket_setting_always">Always</string>
+    <string name="websocket_persistent_notification">In order to maintain the websocket connection the app will need to create a persistent notification. You may use the button below to manage the appearance of this notification. It is recommended to minimize the notification to hide it.</string>
```

Comment:
I think most people will expect that when you say "to hide it", the notification will completely disappear, whereas minimizing it will (on most devices) only remove the icon from the status bar. Suggestion: change to "to hide the icon" or suggest disabling the category which will actually hide the notification.

Reply:
For troubleshooting issues its best for us to ask for it to be minimized in case there are issues with the foreground service. I will update the text though to make it more clear.